app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+
+class Carnivore(Animal):
+    def check_is_alive(self, animal: Animal) -> None:
+        if animal.health <= 0:
+            self.alive.remove(animal)
```

Comment:
Better to move this to Animal class

Reply:
And this method need to check that instance alive or not(not parameter)

Reply:
@ArsenPidhoretskyi Do you mean that i also have to return true or false?

Reply:
Oh i'm sorry, i accidentally requested review, i just wanted to ask question above
@ArsenPidhoretskyi

Reply:
No you need to move this method to class Animal
It must accept only self, not animal because it is so strange